### PR TITLE
PP-9793 webhooks smoke test

### PR DESCRIPTION
**Pull request by nlsteers**
## What?

Added a new step to the existing payment smoke tests to validate that webhooks are received

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,41 +14,45 @@ The scripts which run the Canaries post application deploy can be found in [pay-
...
+| make-card-payment-sandbox-without-3ds          | production  | card_sandbox_prod     |
+| make-card-payment-stripe-with-3ds2             | production  | card_stripe_3ds_prod  |
+| make-card-payment-stripe-without-3ds           | production  | card_stripe_prod      |
+| make-card-payment-worldpay-with-3ds            | production  | card_wpay_3ds_prod    |
+| make-card-payment-worldpay-with-3ds2           | production  | card_wpay_3ds2_prod   |
+| make-card-payment-worldpay-with-3ds2-exemption | production  | card_wpay_3ds2ex_prod |
+| make-card-payment-worldpay-without-3ds         | production  | card_wpay_prod        |
+| notifications-sandbox                          | production  | notifications_prod    |
+| cancel-card-payment-sandbox-without-3ds        | production  | cancel_sandbox_prod   |
+| use-payment-link-for-sandbox                   | production  | pymntlnk_sandbox_prod |
```
Comment (katstevens) [inline README.md:48]: This is much easier to read btw - thanks!

File: make-card-payment-worldpay-without-3ds/index.js [hunk 2]
```diff
@@ -32,4 +32,5 @@ exports.handler = async () => {
   if (paymentStatus !== 'success') {
     throw new Error(`Payment status ${paymentStatus} does not equal success`)
   }
+  if (WEBHOOKS_ENABLED === 'true') await smokeTestHelpers.validateWebhookReceived(ENVIRONMENT, payment.payment_id)
```
Comment (katstevens) [inline make-card-payment-worldpay-without-3ds/index.js:35]: What does the failure path look like for this? e.g if we're looking at a failed result in the Concourse output, will it be obvious that it's due to webhooks?

**Review by katstevens — COMMENTED** [review]
Looking good - one questions about the unhappy path

File: make-card-payment-worldpay-without-3ds/index.js (hunk 2 above)
Comment (nlsteers) [reply make-card-payment-worldpay-without-3ds/index.js:35]: The validate helper function logs some webhooks specific things for clarity.

If the promise returned by the s3 function is rejected that is also a fairly obvious error message (e.g. "key not found"). This also fails the test.

Comment (katstevens) [reply make-card-payment-worldpay-without-3ds/index.js:35]: Cool - if S3 is returning a sensible error message that should be fine 👍

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
   "devDependencies": {
     "path": "^0.12.7",
     "proxyquire": "^2.1.3",
-    "standard": "^17.0.0",
+    "standard": "~14.3.x",
```
Comment (katstevens) [inline package.json:21]: Oh sorry one more question - why did we have to downgrade `standard`?

Comment (nlsteers) [reply package.json:21]: Mainly because this is the version we use everywhere else (and also because it was breaking my linter lol)

Comment (katstevens) [reply package.json:21]: Haha ok. I think everywhere else should probably be brought up to speed with this, but it's not a blocker.

**Pull request merged**